code/modules/mob/living/carbon/hallucinations.dm

```diff
@@ -233,11 +253,8 @@
 		I.color = COLOR_BLOOD_HUMAN
 		return I
 	else
-		var/image/I = image('icons/obj/ammo.dmi', "s-casing-spent", layer = BELOW_TABLE_LAYER)
-		I.layer = BELOW_TABLE_LAYER
-		I.dir = pick(GLOB.alldirs)
-		I.pixel_x = rand(-10,10)
-		I.pixel_y = rand(-10,10)
+		var/image/I = image('icons/effects/blood.dmi', "gib[pick("arm", "leg", "torso")]_flesh", layer = BELOW_TABLE_LAYER)
+		I.color = WOOD_COLOR_PALE
```

Comment:
planes + layers for images are important

Reply:
layer is set in constructor :/